1config-ui/src/com/brunobonacci/oneconfig/ui/controller.cljs

```diff
@@ -1,6 +1,8 @@
 (ns com.brunobonacci.oneconfig.ui.controller
+  (:require-macros [cljs.core.async.macros :refer [go]])
   (:require
    [reagent.core :as reagent :refer [atom]]
+   [cljs.core.async :as async :refer [<! put! chan]]
```

Comment:
Although I believe there is no need for `core.async` if you want to use it, we need to add the dependency in the `project.clj`

Reply:
sure but I'll check if we really need it